references/classification/presets.py

```diff
@@ -13,18 +13,25 @@ def __init__(
         interpolation=InterpolationMode.BILINEAR,
         hflip_prob=0.5,
         auto_augment_policy=None,
+        ra_magnitude=9,
+        augmix_severity=3,
         random_erase_prob=0.0,
+        center_crop=False,
     ):
-        trans = [transforms.RandomResizedCrop(crop_size, interpolation=interpolation)]
+        trans = (
+            [transforms.RandomResizedCrop(crop_size, interpolation=interpolation)]
+            if center_crop
+            else [transforms.CenterCrop(crop_size)]
```

Comment:
@TeodorPoncu I think this is a bug. I believe you meant to write:
```python
            [transforms.CenterCrop(crop_size)]
            if center_crop
            else [transforms.RandomResizedCrop(crop_size, interpolation=interpolation)]
```

Could you please confirm?

Reply:
Yes, sorry for that. You've correctly guessed what I wanted to write. Thanks for catching it out. I think the `--train-center-crop` flag should be removed from the docs as well to reflect the way the weights were trained.

Reply:
@TeodorPoncu thanks for coming back to me. Does this mean that you didn't actually use the flag during training? Can we remove it?

Reply:
Yes, the flag can be removed.

Reply:
That's not what I see on the training log you used on the trained model. I see that `train_center_crop=True`. Do we have the right model available on the checkpoint area on AWS?

Reply:
Yes, but given the fix in order to replicate, one will have to run with `train_center_crop=True` in order to have the same preprocessing behavior as the AWS weights had.

Reply:
Yes, the bug was introduced and used when performing the training as in 1fddecc8d4e3cadafa33993dc82b0f82b93a8dd7

Reply:
Agreed. I checked all commits prior f561edfc8f012425cb30986439eaacee2695e3cf (date before training) and all of them use RandomCrop. I'll remove the flag.